packages/reserve-site/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "celoreserve.org",
+  "version": "1.0.0",
+  "private": true,
+  "scripts": {
+    "dev": "next dev",
+    "build": "yarn tsc",
+    "prepare": "next build",
+    "deploy": "yarn run kill-dev; yarn run clean-next-cache; yarn run prepare; gcloud app deploy $MANIFEST",
+    "deploy:dev": "gcloud config set project celo-reserve-site-281118; NODE_ENV=production DEPLOY_ENV=development MANIFEST=development.yaml yarn run deploy && yarn view",
+    "deploy:prod": "gcloud config set project celo-reserve-site-281118; NODE_ENV=production DEPLOY_ENV=production MANIFEST=production.yaml yarn run deploy && yarn view -s production",
+    "view": "gcloud app browse",
+    "kill-dev": "kill -9 $(lsof -ti :3000)",
+    "clean-next-cache": "rm -rf .next",
+    "start": "next start -p $PORT"
+  },
+  "dependencies": {
+    "@emotion/core": "^10.0.28",
+    "airtable": "^0.8.1",
+    "front-matter": "^4.0.2",
+    "markdown-to-jsx": "^6.11.4",
+    "next": "9.4.4",
+    "raw-loader": "^4.0.1",
+    "react": "16.13.1",
+    "react-dom": "16.13.1"
```

Comment:
I noticed there's a *lot* of churn in the yarn.lock file, probably because these don't match what we use in other packages like web. Would it be difficult to match the current versions and/or upgrade web? Not mandatory but I've found that things can get funky in yarn workspaces when versions differ.

Reply:
downgraded to match web package